data/mods/PK_Rebalance v0.75/pk_critter_zombie.json

```diff
@@ -0,0 +1,939 @@
+    "armor_fire":3,
+    "vision_day":50,
+    "vision_night":10,
+    "death_drops": "default_zombie_death_drops",
+    "hp":180,
+    "death_function":["NORMAL", "BLOBSPLIT"],
+    "special_attacks":[["UPGRADE", 30], ["RESURRECT", 30], ["FORMBLOB", 180], ["LONGSWIPE", 10]],
+    "description":"Standing with errect back and mocking smile, this graceful zombie appears to actually direct nearby zombies with glares and the twitching of its jagged claws.  Its skin is a revolting shade of pink, and every once in a while it stops what it is doing to excrete a few gooey, animate blobs.",
+    "flags":["SEES", "HEARS", "GOODHEARING", "SMELLS", "WARM", "BASHES", "POISON", "NO_BREATHE", "REVIVES", "BONES"],
+    "anger_triggers":["PLAYER_CLOSE", "PLAYER_WEAK"],
+    "fear_triggers":["HURT"]
+    },{
+    "type" : "MONSTER",
+    "id" : "mon_zombie_shady_pk",
+    "name": "ethereal zombie",
+    "species":"ZOMBIE",
+    "default_faction":"zombie",
+    "symbol":"Z",
+    "color":"black_white",
+    "size":"MEDIUM",
+    "material":"flesh",
+    "diff":16,
+    "aggression":50,
+    "morale":90,
+    "speed":100,
+    "melee_skill":7,
+    "melee_dice":3,
+    "melee_dice_sides":8,
+    "melee_cut":3,
+    "dodge":5,
+    "armor_bash":8,
+    "armor_cut":15,
+    "armor_fire":5,
+    "vision_day":25,
+    "vision_night":50,
+    "hp":90,
+    "death_function":"NORMAL",
+    "special_attacks":[{
+            "type" : "bite",
+            "cooldown" : 5
+        }, ["GRAB", 4]],
+    "description":"The permanent shadow around this gaunt figure makes it hard to make out.  What you can see appears translucent and shimmering.  It is disoriented in light, and actively avoids fire.",
+    "flags":["HARDTOSHOOT", "SEES", "HEARS", "SMELLS", "STUMBLES", "WARM", "BASHES", "GROUP_BASH", "POISON", "BLEED", "NO_BREATHE", "REVIVES", "BONES", "PUSH_MON", "NIGHT_INVISIBILITY", "SWIMS"],
+    "categories":["CLASSIC"],
+    "anger_triggers":["PLAYER_CLOSE", "FRIEND_DIED", "STALK", "HURT"],
+    "placate_triggers":["FIRE"]
+    },{
+    "type" : "MONSTER",
+    "id" : "mon_zombie_hulk_pk",
+    "name": "zombie tank",
+    "species":"ZOMBIE",
+    "default_faction":"zombie",
+    "symbol":"Z",
+    "color":"magenta_white",
+    "size":"HUGE",
+    "material":"flesh",
+    "diff":60,
+    "aggression":100,
+    "morale":100,
+    "speed":150,
+    "melee_skill":7,
+    "melee_dice":2,
+    "melee_dice_sides":20,
+    "melee_cut":5,
+    "dodge":4,
+    "armor_bash":20,
+    "armor_cut":20,
+    "armor_stab":10,
+    "armor_fire": 5,
+    "vision_day":50,
+    "vision_night":4,
+    "death_drops": "mon_zombie_hulk_death_drops",
+    "hp":666,
+    "death_function":["NORMAL"],
+    "special_attacks":[["SMASH", 20], ["BIO_OP_TAKEDOWN", 30]],
+    "description":"Rumored to be the conjoining of several bodies, the massive Zombie Tank is perhaps the ultimate tool of destruction.  Making up for what it lacks in skill with brutality, it is the demolisions of the zombie scourge.",

```

Comment:
"it is the demolisions of the zombie scourge" - doesn't sound right. "Wrecking ball" maybe? "One-zombie demolitions squad"?